magento/magento2
# Implemented UrlFilterApplier component

### Description (*)
Implemented a new component to apply filters on a grid via GET URL parameter. Added the component to the product, cms_page, and cms_block grids.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1501

### Manual testing scenarios (*)

1. Open one of the grids mentioned above
2. Add the filters parameter to the URL, example for product: ?filters[name]=product123
3. The grid should be loaded with a name active filter with value = product123

### Questions or comments


### Contribution checklist (*)
